### DISCUSSION TITLE
Issue while importing employees excel sheet

### DISCUSSION THREAD

**Discussion by sharukhkhan234** [horilla-opensource/horilla#244] [Q&A]
Hi!
I tried so many times to import the employee's Excel sheet which contains probably 20 employees, I filled the sheet as the template provided by the software, which means the entries that I have put in the Excel sheet are in the import template. now the issue that I am facing is that can't import the Excel sheet, On the first attempt it generated an error that the template doesn't match with the existing template, and on the next try it generated an error (the user with this email ID already exists). But actually, no such emails are showing in the software. Before importing so many employees' Excel sheets, I tried to import a single employee, and it imported correctly. Now my question is! where the email IDs are stored when the sheet is not imported.

**Comment by 86-ark** [comment]
Greetings, 
Dear   sharukhkhan234, My name is Abdulrezak From Ethiopia, I installed Horilla HRMS but I can't connect with mysql DB Please Help me

**Reply by sharukhkhan234** [reply]
Greetings,
Can you share a snip from the databases section in the settings.py file

**Reply by 86-ark** [reply]
![Capture](https://github.com/user-attachments/assets/3892c497-f738-45de-97aa-cfea2e5d6c15)

**Reply by sharukhkhan234** [reply]
did you install mysql? If not, plz install it first and make a database, and remember the credentials. or If you have already installed then try this..

1. pip install mysqlclient  # (in the command line)
2.  DATABASES = {
	'default': {
		'ENGINE': 'django.db.backends.mysql',
		'NAME': 'mydb',
		'USER': 'root',
		'PASSWORD': 'admin',
		'HOST':'localhost',
		'PORT':'2024', 
	}
}

make sure the correct ( name, user, password, host, port). put the details above as you have given to the database.

3. python manage.py makmigrations
4. python manage.py migrate
5. python manage.py runserver

**Comment by horilla-opensource** [comment]
> Hi! I tried so many times to import the employee's Excel sheet which contains probably 20 employees, I filled the sheet as the template provided by the software, which means the entries that I have put in the Excel sheet are in the import template. now the issue that I am facing is that can't import the Excel sheet, On the first attempt it generated an error that the template doesn't match with the existing template, and on the next try it generated an error (the user with this email ID already exists). But actually, no such emails are showing in the software. Before importing so many employees' Excel sheets, I tried to import a single employee, and it imported correctly. Now my question is! where the email IDs are stored when the sheet is not imported.

Hi @sharukhkhan234 , 
Did you receive the error report excel sheet showing the errors in import?

**Reply by sharukhkhan234** [reply]
Hi!
Resolved

**Reply by horilla-opensource** [reply]
Hi @sharukhkhan234 ,
Great.
Also please do care to use the issue section of the repo for issue related queries and discussion topics here.
Sometimes the discussion notifications are missed from our end, that's why the late reply. Sorry for that.

With Regards,
**Team Horilla**
